homework/index.js

```diff
@@ -30,13 +30,127 @@
     return elem;
   }
 
+  function buildRepositoryInfo(repositoryObject, containerDiv) {
+    const repositoryTable = createAndAppend('table', containerDiv, {
+      id: 'repository-table',
+    });
+    const repositoryTableBody = createAndAppend('tbody', repositoryTable, {});
```

Comment:
If you don't want to set any attributes with the third parameter of `createAndAppend` then there is no need to specify an empty object for this parameter. The default for this parameter is already an empty object:

```js
function createAndAppend(name, containerDiv, options = {}) {
```

The ES6 syntax `options = {}` means that the default for the `options` parameter if not specified is (in this case) an empty object.